Matlab.tmbundle/Syntaxes/MATLAB.tmLanguage

```diff
@@ -1543,7 +1543,7 @@
 									<key>name</key>
 									<string>meta.parameters.matlab</string>
 									<key>begin</key>
-									<string>(?&lt;=[a-zA-Z0-9_])\s*\(</string>
+									<string>\s*\(</string>
```

Comment:
Previous regex expected `=` and function name to start on the same line as `(`. This is not the case anymore by adding line continuations to the `meta.parameters.matlab` scope below.